extensions/shared/wrap-paid-block/style.scss

```diff
@@ -1,8 +1,10 @@
-.jetpack-paid-block__wrapper {
-	border: 1px solid var( --color-border-subtle );
-}
+@import '../styles/gutenberg-colors.scss';
+@import '../styles/gutenberg-variables.scss';
 
-.jetpack-paid-block__disabled {
-	opacity: 0.5;
-	padding: 10px;
+.block-editor-block-list__block:not( .is-selected ) .jetpack-paid-block__wrapper {
+	border: $border-width solid $light-gray-500; // To reflect the `Warning` component from `@wordpress/editor`
+	margin-left: -14px;
+	margin-right: -14px;
+	padding-left: calc( 14px - #{$border-width} );
+	padding-right: calc( 14px - #{$border-width} );
```

Comment:
14px might be good candidate for a variable: https://github.com/WordPress/gutenberg/blob/9052d4c26fff89907d7e7abec745a3d0079dd178/assets/stylesheets/_variables.scss#L57

Reply:
6ecaace